.github/workflows/push.yml

```
@@ -447,57 +487,120 @@
+        id: read
+        with:
+          matrix-step-name: docker-build
+
+      - name: Setup yarn
+        uses: ./monorepo/.github/actions/setup-yarn
+        with:
+          working-directory: monorepo
+          RUNS_ON_S3_BUCKET_CACHE: ${{ secrets.S3_DOCKER_CACHE_BUCKET }}
+
+      - name: Install infra modules
+        if: steps.infra-node-modules.outputs.cache-hit != 'true'
```

actionlint (1.7.4)

property "infra-node-modules" is not defined in object type {read: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)